### DISCUSSION TITLE
SubjectAltName

### DISCUSSION THREAD

**Discussion by pragya1998hal** [Keyfactor/ejbca-ce#756] [General]
Hello sir 

I am using CMP configuration and selected end-entity certificate as CMP Authentication Module.

My issue is the end-entity certificate I provided in the extraCerts field already contains the required Subject Alternative Names (SAN) with the correct IP address and DNS name.

Still the new certificate which is created in that Subject Alternative Names (SAN) field IP address and DNS name is missing.

Is there any way or possibility that we can override Subject Alternative Names filed values in new certificate with values of Subject Alternative Names passed in end-entity certificate I provided in the extraCerts.

Or in CMP request we need to include the required SANs in the SubjectAltName field.

**Comment by primetomas** [comment]
Requested fields always comes from the request. Authentication certificates (extraCerts) are only used for authentication and are not considered as a request for any fields.
